src/templates/tabs/trust.jade

```diff
@@ -331,14 +331,28 @@ section.col-xs-12.content(ng-controller="TrustCtrl")
                     .col-sm-2.col-md-2.inputPadding
                       div.rippling-checkbox
                         label(l10n) Rippling&#32;
-                          input(name='rippling', type='checkbox', ng-model='trust.rippling', ng-disabled='trust.balance !== "0" && isIncomingOnly()')
+                          input(name='rippling', type='checkbox', ng-model='trust.rippling', ng-disabled='trust.balance !== "0" && isIncomingOnly() && ripplingEnabled()')
                     .col-sm-7.col-md-6
                       .row.row-padding-small
-                        .col-xs-6.col-sm-5(ng-hide="trust.loading && trust.state==='removing'")
-                          button.btn.btn-block.btn-primary#save(type='submit'
-                            ng-disabled='accountForm.$invalid || trust.loading')
-                            span(ng-hide="trust.loading", l10n) Save
-                            span(ng-show="trust.loading && trust.state==='saving'", l10n) Saving...
+                        div(ng-hide="trust.loading && trust.state==='removing'")
+                          .col-xs-6.col-sm-5(ng-hide='trust.balance !== "0" && isIncomingOnly() && trust.rippling && !ripplingEnabled()')
```

Comment:
It's very hard to read so many chained conditions. Can you make a separate function for this?